src/chrome/content/rules/World-Nuclear-News.org.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+	Connection timed out:
+		- admin
+		- dr
+		- wnaweb
```

Comment:
These targets look like internal subdomains and also time out over HTTP so I think we should leave them out entirely

Reply:
@Bisaloo Done.